nativescript-core/ui/core/view/view-common.ts

```diff
@@ -819,9 +819,9 @@ export abstract class ViewCommon extends ViewBase implements ViewDefinition {
         }
     }
 
-    public requestLayout(): void {
+    public requestLayout(calledFromChild): void {
```

Comment:
it should have a default value here as in the view-base
```suggestion
    public requestLayout(calledFromChild = false): void {
```